README.md

```diff
@@ -1,42 +1,38 @@
 # ROCm for Arch Linux
 This repository hosts a collection of [Arch Linux](https://www.archlinux.org/) [PKGBUILDs](https://wiki.archlinux.org/index.php/PKGBUILD) for the [AMD ROCm Platform](https://rocmdocs.amd.com/en/latest/).
-These scripts implement a great portion of the stack, ranging from low level interfaces, over compilers and high-level application librariers.
+These scripts implement a great portion of the stack, ranging from low level interfaces, over compilers to high-level application librariers.
 
 ## Installation
 The Arch Linux packages for ROCm are available on the [AUR](https://wiki.archlinux.org/index.php/Arch_User_Repository).
-Since many packages will be installed, it is recommended to use an [AUR helper](https://wiki.archlinux.org/index.php/AUR_helpers)
-like [`yay`](https://aur.archlinux.org/packages/yay/).
-
-It is also recommended to use the [`arch4edu`](https://wiki.archlinux.org/index.php/Unofficial_user_repositories#arch4edu) binary repository as it will
+Since many packages will be installed, we recommend to use the [`arch4edu`](https://wiki.archlinux.org/index.php/Unofficial_user_repositories#arch4edu) binary repository as it will
 greatly speed up your installation time.
 For directions see [Add arch4edu to your Archlinux](https://github.com/arch4edu/arch4edu/wiki/Add-arch4edu-to-your-Archlinux).
 
-To install ROCm, use the following command:
-```
-yay -S rocm-dkms
+To install ROCm, use
+```bash
+# pacman -Syu rocm-dev rocm-utils rocm-libs
 ```
-> **Warning**: Building the compiler `llvm-amdgpu` needs a lot of RAM (over 40 GiB on 12 threads) and disk space (more than 50 GiB). We recommend to use a [swap file](https://wiki.archlinux.org/index.php/swap#Swap_file).
 
 You can also install specific ROCm packages like so:
-```
-yay -S rocminfo
+```bash
+# pacman -S rocminfo
```

Comment:
Ah one more thing I don't like using shell prefixes like `#` and `$` because then you can't just copy paste the line.

Reply:
Yeah, people should know that you install packages with root privileges.